ansible/roles/vm_set/tasks/docker.yml

```diff
@@ -28,3 +28,13 @@
   apt: pkg=docker-ce update_cache=yes
   become: yes
   environment: "{{ proxy_env | default({}) }}"
+
+- name: remove old python packages
```

Comment:
@lguohan please have a look

Reply:
@msosyak This should be aligned with the 201811 branch: added a similar steps to install missing docker-py library or backported new docker api calls.

Reply:
@nazariig  Same step is present on 201811 branch
https://github.com/Azure/sonic-mgmt/blob/9416328ebbb7594dec8bdfdfbba6a560693f2dd5/ansible/roles/vm_set/tasks/docker.yml#L32

Reply:
@msosyak IMHO, a conflict with the new version is possible since remove logic is missing.

Reply:
@nazariig actually installation of docker-py 1.7.2 was removed in https://github.com/Azure/sonic-mgmt/pull/1213 and I have tried to run a version of mgmt scripts from the old container on the same setup after run this changes and did not face any issue with docker module.

Instead, I wasn't able to load the docker module after old scripts run, this is why I added remove and forcereinstall step.

Reply:
@msosyak ok, then no concerns.